content/docs/guides/gke/troubleshooting-gke.md

```diff
@@ -24,6 +24,64 @@ Here are some tips for troubleshooting Cloud IAP.
 
  * Make sure you are using HTTPS
 
+### DNS name not registered
+
+This section provides troubleshooting information for problems creating a DNS entry for your ingress. This section assumes
```

Comment:
This is the first time we use the term "ingress" on this page. It'd be good to give more context about what we mean by "ingress".